org.osgi.test.common/src/main/java/org/osgi/test/common/annotation/InjectBundle.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) OSGi Alliance (2020). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.osgi.test.common.annotation;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Inherited
+@Target({
+	FIELD, PARAMETER
+})
+@Retention(RUNTIME)
+@Documented
+public @interface InjectBundle {
+
+	public enum Type {
+		EMBEDDED,
+		URL
+	}
+
+	Type type() default Type.EMBEDDED;
+
+	String location();
```

Comment:
Why not just `value()` so that the simple case is just `@InjectBundle(TB1_JAR)`? At worst if you use any of the other properties you still have less chars to type and `location` isn't that much better than `value` given even a single line of javadoc.

Reply:
renamed it